Patches/IntroPatch.cs

```diff
@@ -445,12 +456,9 @@ public static void Postfix(IntroCutscene __instance)
                 PlayerControl.LocalPlayer.Data.Role.IntroSound = DestroyableSingleton<HudManager>.Instance.TaskCompleteSound;
                 break;
 
-            case CustomRoles.Sheriff:
-            case CustomRoles.Veteran:
```

Comment:
any necessary reason for changing this?

Reply:
Yeah I agree Veteran, Sheriff, and Nice Guesser intro sound shouldn’t be changed

Reply:
When you as an unmodded client get Sheriff, do you see "Crewmate" or "Impostor"?

Reply:
> When you as an unmodded client get Sheriff, do you see "Crewmate" or "Impostor"?

sheriff?